postgresql.yaml

```diff
@@ -0,0 +1,38 @@
+image:
+  repository: dhis2/postgresql-curl
+  pullPolicy: IfNotPresent
+  tag: 13
+
+primary:
+  initdb:
+    scripts:
+      seed.sh: |
+        #!/usr/bin/env bash
+
+        set -euo pipefail
+
+        export PGPASSWORD=$POSTGRES_POSTGRES_PASSWORD
+
+        function exec_psql() {
+          psql -U postgres -qAt -d dhis2 -c "$1"
+        }
+
+        SEED_URL="https://databases.dhis2.org/sierra-leone/2.39.0/dhis2-db-sierra-leone.sql.gz"
```

Comment:
This should be the [2.39.1 SL DB](https://databases.dhis2.org/sierra-leone/2.39.1/dhis2-db-sierra-leone.sql.gz).